### ISSUE TITLE
Suggestion: GameObject.replaceBodyType()

### ISSUE THREAD

**Issue by rafcolson** [GoranM/bdx#305]
Currently there's only ```dynamic(boolean)``` to change body types. And it isn't exactly perfect. Because if you would want to change RIGID_BODY to DYNAMIC, this method would do the trick. Actually, this method could be better described by ```suspendDynamics()``` and ```restoreDynamics()```, which I think should be private methods.

Then there could be methods to actually change the body type:
-```noCollision(boolean) to switch between "NO_COLLISION" and "STATIC"
- ```dynamic(boolean)``` (to replace the old one) to switch between "STATIC" and "DYNAMIC"
- ```rigidBody(boolean)``` to switch between "DYNAMIC" and "RIGID_BODY"

Of course "SENSOR" should be treated as "STATIC" and so I propose to change this in a new ```bodyType``` variable at initialization, which will be derived from ```json.get("physics").get("body_type")```. Also I think it would be better to change to small caps.

These three methods would be mere convenience functions. I think we would benefit more from one method to replace the body type, so: ```replaceBodyType(String)```.

**Comment by GoranM** [comment]
A `bodyType(String)` would be fine. With that, there should be no need for the other methods you mentioned.

> Of course "SENSOR" should be treated as "STATIC" and so I propose to change this in a new bodyType variable at initialization, which will be derived from json.get("physics").get("body_type"). Also I think it would be better to change to small caps.

Well, 'SENSOR' indicates a 'STATIC' object set to ghost, so it's a relevant distinction. In general, I think we should store blender data as exported, and then use it in BDX according to our actual needs.

It also makes sense to keep blender notation for body types, which is capital case - It also hints that those strings are used as flags, rather than just arbitrary data.

**Comment by rafcolson** [comment]
Alright, so I'll have a go at this. I'll submit a PR after #306 and another PR that would add support for joined masses with compound objects. It would be nice if this first got in so I immediately could use the ```mass()``` method in some applications. To be honest, I already have a rough commit with changes to mass, dynamics and parent.

**Comment by rafcolson** [comment]
PR submitted.

**Comment by rafcolson** [comment]
I'll have another go at this and hope I'll get it right the first time.

**Comment by rafcolson** [comment]
PR submitted.

**Comment by rafcolson** [comment]
PR merged.
